docker-compose.yaml

```diff
@@ -22,12 +34,12 @@ services:
   generator:
     image: killrvideo/killrvideo-generator:1.2.4
     depends_on:
-    - dse
-    - etcd
+      - dse
+      - etcd
     environment:
       KILLRVIDEO_ETCD: "etcd:2379"
-      KILLRVIDEO_DSE_USERNAME: $KILLRVIDEO_DSE_USERNAME
-      KILLRVIDEO_DSE_PASSWORD: $KILLRVIDEO_DSE_PASSWORD
+      KILLRVIDEO_DSE_USERNAME: cassandra
+      KILLRVIDEO_DSE_PASSWORD: cassandra
```

Comment:
Same as above.  The value for KILLRVIDEO_DSE_PASSWORD cannot be set to the default